### PR TITLE
More unit tests and modfied conftest

### PR DESCRIPTION
**Pull request by Hofer-Julian**
- test update, get_current_time and get_value_ptr
- make flopy_dis more special to catch edge cases

Please see https://issuetracker.deltares.nl/browse/IMOD6-284

### CODE REVIEW

File: tests/test_unit_modflow_dis.py [hunk 1]
```diff
@@ -96,3 +96,87 @@ def test_get_end_time(flopy_dis, modflow_lib_path):
...
+    model = sim.get_model(sim.name)
+    chd = model.get_package("chd_0")
+    stress_period_data = chd.stress_period_data.array[0]
+
+    # TODO: Refactor this as soon as `get_var_grid` is implemented
+    dis = model.get_package("dis")
+    len_column = len(dis.delr.array)
+
+    mf6.update()
+    actual_head = mf6.get_value_ptr("SLN_1/X")
```
Comment (mjr-deltares) [inline tests/test_unit_modflow_dis.py:176]: let's try and use <model_name>/X, which should also work now. Shortly we will be able to even use <model_name>/head (and <model_name>/concentration for transport) which is preferable I guess
